### PR TITLE
Spec trigger attestation

### PR DESCRIPTION
**Pull request by csharrison**
This doesn't currently cover the token issuance steps at request time. Those will require fetch monkey patches.

***
<a href="https://pr-preview.s3.amazonaws.com/WICG/attribution-reporting-api/pull/616.html" title="Last updated on Nov 14, 2022, 9:36 PM UTC (963007e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/attribution-reporting-api/616/1d5f4b7...963007e.html" title="Last updated on Nov 14, 2022, 9:36 PM UTC (963007e)">Diff</a>

### CODE REVIEW

File: index.bs
Comment (apasel422) [inline index.bs:258]:
```suggestion
:: A [=byte sequence=].
```

File: index.bs
Comment (apasel422) [inline index.bs:332]:
```suggestion
:: A [=byte sequence=].
```

File: index.bs
Comment (apasel422) [inline index.bs:1067]:
```suggestion
1. Let |decoded| be the result of [=forgiving-base64 decoding=] |encodedBlindedPrivateStateToken|.
```

File: index.bs
Comment (apasel422) [inline index.bs:1075]:
```suggestion
```

File: index.bs
Comment (apasel422) [inline index.bs:1077]:
```suggestion
    the empty [=byte sequence=] (data), and 0 (the null timestamp).
```

File: index.bs
Comment (apasel422) [inline index.bs:1081]:
```suggestion
1. Return the result of [=forgiving-base64 encoding=] |redeemedBytes|.
```

File: index.bs
Comment (apasel422) [inline index.bs:1083]:
```suggestion
```

File: index.bs
Comment (apasel422) [inline index.bs:1734]:
```suggestion
        [=header list/append=] a new [=header=] named "`Sec-Attribution-Reporting-Private-State-Token`" to |newHeaders|
```

File: index.bs
Comment (apasel422) [inline index.bs:1736]:
```suggestion
1. Let |request| be the result of executing [=create a report request=] on |url|, |data|, and |newHeaders|.
```

File: index.bs
Comment (apasel422) [inline index.bs:1688]: We should default `newHeaders` to the empty list, or else some other invocations of itthis algorithm in the spec don't "type check".

**Review by apasel422 — CHANGES_REQUESTED** [review]

File: index.bs
Comment (csharrison) [reply index.bs]: Done

**Review by apasel422 — APPROVED** [review]

**Comment by csharrison** [comment]
Will wait for @aykutbulut's review before landing.

File: index.bs
Comment (aykutbulut) [inline index.bs:1083]: I assume encoded indicates the token is base64 encoded? Can we drop it and simply say privateStateToken?

**Review by aykutbulut — APPROVED** [review]
LGTM. I have a minor comment added for the author to think about.

File: index.bs
Comment (csharrison) [reply index.bs:1083]: sure

**Comment by csharrison** [comment]
Thanks @aykutbulut

**Pull request merged**
